core/src/com/unciv/logic/map/mapunit/MapUnit.kt

```diff
@@ -654,13 +660,15 @@ class MapUnit : IsPartOfGameInfoSerialization {
     }
 
     fun updateUniques() {
-        val uniqueSources =
-                baseUnit.uniqueObjects.asSequence() +
-                        type.uniqueObjects +
-                        promotions.getPromotions().flatMap { it.uniqueObjects } +
-                        statuses.flatMap { it.uniques }
+        val unitUniqueSources =
+            baseUnit.uniqueObjects.asSequence() +
+                type.uniqueObjects
+        val otherUniqueSources = promotions.getPromotions().flatMap { it.uniqueObjects } +
+            statuses.flatMap { it.uniques }
+        val uniqueSources = unitUniqueSources + otherUniqueSources
```

Comment:
What's with assuming that promotions *won't* provide conditional tags? I can totally see modders using this

Reply:
I'm not sure I understand the question. Promotions are being checked here, no?